VideoLocker/src/main/java/org/edx/mobile/view/CourseBaseActivity.java

```diff
@@ -33,14 +34,23 @@
  *  2. progress_spinner
  *  3. offline_mode_message
  */
+@ContentView(R.layout.activity_course_base)
 public abstract  class CourseBaseActivity  extends BaseFragmentActivity implements TaskProcessCallback{
 
-    private View offlineBar;
-    private View lastAccessBar;
-    protected View downloadProgressBar;
-    protected IconView downloadIndicator;
+    @InjectView(R.id.offline_bar)
+    View offlineBar;
 
-    protected ProgressBar progressWheel;
+    @InjectView(R.id.last_access_bar)
+    View lastAccessBar;
+
+    @InjectView(R.id.download_in_progress_bar)
+    View downloadProgressBar;
+
+    @InjectView(R.id.video_download_indicator)
+    TextView downloadIndicator;
```

Comment:
This should be an `IconView`, and the `Iconify.setIcon()` call can be removed. Your branch did not include the icon RTL implementation, so this was a regression in merging.